apis/apis.json

```diff
@@ -586,12 +614,14 @@
     "serviceYaml": "oslogin_v1.yaml",
     "productName": "Google Cloud OS Login API",
     "productUrl": "https://cloud.google.com/compute/docs/instances/managing-instance-access",
-    "version": "1.0.0-beta02",
+    "version": "1.0.0",
     "type": "grpc",
     "description": "Recommended Google client library to manage OS login configuration for Google account users.",
     "tags": [ "OsLogin", "Login" ],
     "dependencies": {
-      "Google.Cloud.OsLogin.Common": "project"
+      "Google.Api.Gax.Grpc": "2.9.0",
+      "Google.Cloud.OsLogin.Common": "project",
```

Comment:
Should this "project" value be a specific version number in a GA release?

Reply:
It can't be at the moment, because Google.Cloud.OsLogin.Common hasn't been released yet. The release tagger checks that both of them are being released at the same time. After 1.0.0 has been released, we can then modify both OsLogin.V1 and OsLogin.V1Beta1 to refer to it.